test/integration/basic_memif_test.go

```diff
@@ -26,8 +27,9 @@ func TestSimpleMemifConnection(t *testing.T) {
 	k8s.PrepareDefault()
 
 	nodes := nsmd_test_utils.SetupNodesConfig(k8s, 1, defaultTimeout, []*pods.NSMgrPodConfig{})
+	useIPv4 := true
```

Comment:
Can you make it configurable? 
This way the tests could run for both IPv4 and 6, when we decide to test IPv6